src/os/ObjectStore.h

```diff
@@ -1371,8 +1371,7 @@ class ObjectStore {
    * has information of src_offset, dest_offset and length.
    * Finally, the source object is deleted.
    */
-  void move_tempobjs(const ghobject_t& src_oid, ghobject_t oid,
-                     const coll_t& cid,
+  void merge_delete_srcobj(const coll_t& cid, const ghobject_t& src_oid, ghobject_t oid,
```

Comment:
Let's call it move_ranges_destroy_src.  Sorry, I should have been more specific.